### PR TITLE
Instruction to install tldr

### PR DESCRIPTION
**Pull request by seisman**
#110

### CODE REVIEW

**Comment by seisman** [comment]
Ping @core-man for review.

File: source/computer/fedora-setup.rst [hunk 1]
```diff
@@ -193,9 +193,18 @@ git
 命令行工具
 ----------
 
-日常科研所需的大多数命令行工具已经默认安装在 Fedora 系统里了。
+日常科研所需的大多数命令行工具已经默认安装在 Fedora 系统里了。这里额外推荐一些
+有用的命令行工具。
 
-这一部分暂无推荐。
+`tldr <https://tldr.sh/>`__ 是一个提供命令的常用用法和示例的命令行工具，
```
Comment (core-man) [inline source/computer/fedora-setup.rst:199]:
```suggestion
tldr
^^^^

`tldr <https://tldr.sh/>`__ 是一个提供命令的常用用法和示例的命令行工具，
```

File: source/computer/ubuntu-setup.rst [hunk 2]
```diff
@@ -203,9 +203,18 @@ git
 命令行工具
 ----------
 
-日常科研所需的大多数命令行工具已经默认安装在 Ubuntu 系统里了。
+日常科研所需的大多数命令行工具已经默认安装在 Ubuntu 系统里了。这里额外推荐一些
+有用的命令行工具。
 
-这一部分暂无推荐。
+`tldr <https://tldr.sh/>`__ 是一个提供命令的常用用法和示例的命令行工具，
```
Comment (core-man) [inline source/computer/ubuntu-setup.rst:209]:
```suggestion
tldr
^^^^

`tldr <https://tldr.sh/>`__ 是一个提供命令的常用用法和示例的命令行工具，
```

File: source/computer/macos-setup.rst [hunk 3]
```diff
@@ -274,6 +274,16 @@ Homebrew 将 GNU 实用工具安装在 :file:`/usr/local/bin` 目录下，但在
 在遇到不兼容的情况下，可以考虑使用 GNU 工具（有前缀 ``g``），但在写脚本时，
 要额外注意脚本的可移植性。
 
+`tldr <https://tldr.sh/>`__ 是一个提供命令的常用用法和示例的命令行工具，
```
Comment (core-man) [inline source/computer/macos-setup.rst:277]: I will submit another PR to list all the commands in macOS.

Comment (seisman) [reply source/computer/macos-setup.rst:277]: Good.

**Review by core-man — APPROVED** [review]
